### ISSUE TITLE
Rails 5.1 support - alias_method_chain removed

### ISSUE THREAD

**Issue by Frexuz** [waymondo/turboboost#24]
Error on app boot:
```
XXX/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/turboboost-0.1.0/lib/turboboost/form_helper.rb:7:in `block in <module:FormHelper>': undefined method `alias_method_chain' for ActionView::Base:Class (NoMethodError)
```

**Comment by waymondo** [comment]
This is fixed on master, though I haven't cut a new version of the gem yet. You can use it with the following: 

```gem 'turboboost', github: 'waymondo/turboboost'```

**Comment by Frexuz** [comment]
Awesome, works :)

**Comment by raycheung** [comment]
@waymondo Any plan to release a new version to include this fix please?

**Comment by waymondo** [comment]
@raycheung done. released version `0.1.1`
